lib/charms/postgresql_k8s/v1/postgresql.py

```diff
@@ -1068,6 +1071,7 @@ def update_user_password(
             with self._connect_to_database(
                 database_host=database_host
             ) as connection, connection.cursor() as cursor:
+                cursor.execute(SQL("RESET ROLE;"))
```

Comment:
PgBouncer needs to reset its user from `pgbouncer_owner` to `relation-X` (which is a superuser) to call `SET LOCAL log_statement = 'none';`.